src/scss/component/accordion/_index.scss

```diff
@@ -0,0 +1,111 @@
+// @gruapl/graupl accordion component styles.
+
+@use "defaults";
+@use "variables" as *;
+@use "../../defaults" as root-defaults;
+@use "../../mixins/layer" as *;
+
+#{defaults.$accordion-selector} {
+  @include layer(component) {
+    border-radius: #{$accordion-border-radius};
+    border: #{$accordion-border};
+
+    #{defaults.$accordion-item-selector} {
+      &>*:first-child {
+        margin: 0;
+
+        #{defaults.$accordion-trigger-selector} {
+          border-style: none;
+        }
+      }
+    }
+
+    #{defaults.$accordion-trigger-selector} {
```

Comment:
This is ultimately a button, lets utilize that fact. If you look at what I did with the alert dismisser: I add the button class to activate a whole of built-in styling I can use.

Reply:
I didn't want to use the button because the styling looked off for an accordion imo. 
Such as the button changing sizes when clicking it.

Reply:
I changed this to a button and I disabled the button size changing when clicked.